client/components/translator-invite/style.scss

```diff
@@ -1,20 +1,32 @@
+@media (max-width: 600px) {
+	.translator-invite {
+		padding-inline-start: 20px;
+	}
+}
+
 .translator-invite__content {
-	margin: 15px auto;
-	font-size: $font-body-extra-small;
-	font-style: italic;
 	max-width: 75%;
-	text-align: center;
+	text-align: start;
+
+	/* stylelint-disable-next-line scales/font-sizes */
+	font-size: 0.9375rem; //typography-exception
+	line-height: 1.3;
+	color: var(--color-text-subtle);
+	margin-block-end: 20px;
 }
 
-.is-section-signup .translator-invite__content,
-.is-section-signup .translator-invite__content a {
-	color: var(--color-text-inverted);
+.translator-invite__heading {
+	/* stylelint-disable-next-line scales/font-sizes */
+	font-size: 1.125rem; //typography-exception
+	font-weight: 500;
+	margin-block-end: 8px;
 }
 
-.is-section-signup .translator-invite__content a {
+.translator-invite__link {
```

Comment:
The font size should be `0.9375rem` and font weight `600` in order to match the styling in the app promo card above.